utils.js

```diff
@@ -1,4 +1,5 @@
 import jwt from "jsonwebtoken";
+
 const generateLogToken = (user) => {
```

Comment:
do usuniecia logowania 
gdzie sa role zapisane w tokenie jak pozniej chcesz walidowac czy to pracownik czy klient ?